samples/samples/snippets_test.py

```diff
@@ -184,8 +193,8 @@ def test_create_database_with_encryption_config(
     assert kms_key_name in out
 
 
-def test_create_database_with_proto_descriptors(capsys, instance_id, database_id):
-    snippets.create_database_with_proto_descriptors(instance_id, database_id)
+def test_create_database_with_proto_descriptor(capsys, instance_id, database_id):
```

Comment:
Where are we writing test for `test_update_database_with_proto_descriptors `?

Reply:
added